recipe/meta.yaml

```diff
@@ -17,25 +17,25 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win]
 
 requirements:
   build:
     - toolchain
     - pkg-config
     - xz 5.2.*
-    - glib
+    - glib 2.51.*
     - libsigcpp
-    - libxml2
-    - glibmm >=2.4
+    - libxml2 2.9.*
+    - glib 2.51.*mm >=2.4
```

Comment:
Ahh great.  I was looking at the current master and it all seemed correct. That would explain it.